pkgs/by-name/pr/prism-model-checker/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  prism-model-checker-unwrapped,
+  java,
+  symlinkJoin,
+  makeWrapper,
+  writeText,
+}:
+let
+  desktopEntry = writeText "prism-model-checker-xprism.desktop" ''
+    [Desktop Entry]
+    Type=Application
+    Icon=prism-model-checker
+    Terminal=false
+    Categories=Science;Math
+    Comment=Probabalistic Symbolic Model Checker
+    Name=XPrism
+    Version=${prism-model-checker-unwrapped.version}
+    Exec=xprism
+  '';
+in
+symlinkJoin {
+  name = "prism-model-checker-${prism-model-checker-unwrapped.version}";
+  nativeBuildInputs = [ makeWrapper ];
+  paths = [
+    prism-model-checker-unwrapped
+    java
```

Comment:
O_O Why does Java need to be symlinked here? Doesn't setting the JAVA_HOME work? If not, a comment would be nice